extensions/amp-jwplayer/amp-jwplayer.md

```diff
@@ -69,27 +69,58 @@ Example:
 </amp-jwplayer>
 ```
 
+Contextual Article Matching
+
+Example:
+
+```html
+  <amp-jwplayer
+      data-player-id="BjcwyK37"
+      data-media-id="BZ6tc0gy"
+      data-content-search="__CONTEXTUAL__"
+      data-content-contextual=true
+      data-content-recency="7D"
+      data-content-backfill=true
+      layout="responsive" width="160" height="90">
+  </amp-jwplayer>
+```
+
 ## Attributes
 
-<table>
-  <tr>
-    <td width="40%"><strong>data-player-id</strong></td>
-    <td>JW Platform player id. This is an 8-digit alphanumeric sequence that can be found in the [Players](https://dashboard.jwplayer.com/#/players) section in your JW Player Dashboard. (**Required**)</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-media-id</strong></td>
-    <td>The JW Platform media id. This is an 8-digit alphanumeric sequence that can be found in the [Content](https://dashboard.jwplayer.com/#/content) section in your JW Player Dashboard. (**Required if `data-playlist-id` is not defined.**)</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-playlist-id</strong></td>
-    <td>The JW Platform playlist id. This is an 8-digit alphanumeric sequence that can be found in the [Playlists](https://dashboard.jwplayer.com/#/content/playlists) section in your JW Player Dashboard.  If both `data-playlist-id` and `data-media-id` are specified, `data-playlist-id` takes precedence.  (**Required if `data-media-id` is not defined.**)</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>common attributes</strong></td>
-    <td>This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.</td>
-  </tr>
-</table>
+##### data-player-id
+
+JW Platform player id. This is an 8-digit alphanumeric sequence that can be found in the [Players](https://dashboard.jwplayer.com/#/players) section in your JW Player Dashboard. (**Required**)
```

Comment:
Attribute sets in docs were recently migrated (#20684) to `<table>` elements. Sorry about that! Would you mind moving this to the new format?